knative-operator/deploy/resources/knativekafka/channel/2-channel-post-install.yaml

```diff
@@ -109,7 +109,7 @@ roleRef:
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: storage-version-migration-kafka-channel-1.1.0
+  generateName: storage-version-migration-kafka-channel-
```

Comment:
Not sure if we should remove the version here. What was the reason we had the version in the name originally?

Reply:
I believe it is to have a different image for each version of this job. 
The problem is that having the version suffix hardcoded forces us to change this at every release (even patch releases).